Library/RemoteConfig/RemoteConfigFeature+HelpersTests.swift

```diff
@@ -40,4 +40,13 @@ final class RemoteConfigFeatureHelpersTests: TestCase {
       XCTAssertFalse(featureFacebookLoginInterstitialEnabled())
     }
   }
+
+  func testCreatorDashboardHidden_RemoteConfig_FeatureFlag_False() {
```

Comment:
Always remember to update the test names to reflect the correct values being tested. In this case we changed `creatorDashboardHiddenEnabled` to `creatorDashboardEnabled` so the test name should change as well.

Reply:
for minor changes like this it could be helpful to comment it as a suggestion so that we can easily commit directly from the PR.

Reply:
Yea, that's normally how I'd do it, but I want us to get into the habit of committing all changes once correctly.
So example - You assign a PR to me, it's reviewed, all changes requested in review, you make as many commits/ask as many questions as you want, but once it's re-assigned back to me, it should pretty much be perfect.

The intention is to reduce the amount of back and forth, just as learning tool. I'm going to explicitly request changes for minor corrections from now on so we get into the habit.